src/app/core/utils/effects.ts

```diff
@@ -0,0 +1,58 @@
+
+import * as rumble from '../../shared/models/rumble';
+import { battleTime } from '../constants/battle';
+
+export const isTeamEffect = (effect: rumble.Effect): boolean => {
+  if (!effect) { return false; }
+  switch (effect.effect) {
+    case 'buff':
```

Comment:
i need penalty because of units like http://optc-db.github.io/characters/#/view/2015
`Inflicts Lv.1 DEF down penalty to crew member(s).`

as for recharge, it will be needed for specials yes... but as of right now i dont think it will make any impact. there is no effect in any ability that is of type recharge, right?

Reply:
Wow, Lucci is a jerk. It's not in the DB yet, but [FN Carrot](http://optc-db.github.io/characters/#/view/1812) does this. We can add it later though